pythran/pythonic/types/float.hpp

```diff
@@ -18,4 +18,57 @@ namespace pythonic
   }
 }
 
+#ifdef ENABLE_PYTHON_MODULE
+
+#include "pythonic/python/core.hpp"
+
+namespace pythonic
+{
+
+  template <>
+  struct to_python<double> {
+    static PyObject *convert(double d)
+    {
+      return PyFloat_FromDouble(d);
+    }
+  };
+  template <>
+  struct to_python<float> {
+    static PyObject *convert(float d)
+    {
+      return PyFloat_FromDouble(d);
+    }
+  };
+  template <>
+  struct from_python<double> {
+    static bool is_convertible(PyObject *obj)
+    {
+      return PyFloat_Check(obj) or PyInt_Check(obj);
+    }
+    static double convert(PyObject *obj)
+    {
+      if (PyFloat_Check(obj))
+        return PyFloat_AsDouble(obj);
+      else
+        return PyInt_AsLong(obj);
+    }
+  };
+  template <>
+  struct from_python<float> {
+    static bool is_convertible(PyObject *obj)
+    {
+      return PyFloat_Check(obj) or PyInt_Check(obj);
+    }
+    static float convert(PyObject *obj)
+    {
+      if (PyFloat_Check(obj))
+        return PyFloat_AsDouble(obj);
+      else
+        return PyInt_AsLong(obj);
+    }
+  };
+}
+
+#endif
+
```

Comment:
int16 is "easy" but `numpy.int64` and `int` are sames in practice but have different types in Python.